controllers/comments.controller.js

```diff
@@ -35,3 +36,16 @@ exports.postCommentWithArticleID = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.deleteComment = async (req, res, next) => {
+  const { comment_id } = req.params
+
+  try {
+    await checkExists("comments", "comment_id", comment_id)
```

Comment:
You could get away without this by checking the return of the delete SQL command and seeing if anything was deleted. But this is fine too.